its/tests/src/test/java/its/ConnectedModeExcludeByVersionTest.java

```diff
@@ -1,119 +0,0 @@
-/*
- * SonarLint Core - ITs - Tests
- * Copyright (C) 2016-2022 SonarSource SA
- * mailto:info AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-package its;
-
-import com.sonar.orchestrator.Orchestrator;
-import com.sonar.orchestrator.locator.MavenLocation;
-import its.tools.OrchestratorUtils;
-import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.ExecutionException;
-import java.util.function.Consumer;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ExpectedException;
-import org.junit.rules.TemporaryFolder;
-import org.sonarqube.ws.client.users.CreateRequest;
-import org.sonarsource.sonarlint.core.ConnectedSonarLintEngineImpl;
-import org.sonarsource.sonarlint.core.client.api.common.PluginDetails;
-import org.sonarsource.sonarlint.core.client.api.connected.ConnectedGlobalConfiguration;
-import org.sonarsource.sonarlint.core.client.api.connected.ConnectedSonarLintEngine;
-import org.sonarsource.sonarlint.core.client.api.connected.ConnectionValidator;
-import org.sonarsource.sonarlint.core.commons.Language;
-import org.sonarsource.sonarlint.core.serverapi.ServerApiHelper;
-
-import static its.tools.ItUtils.SONAR_VERSION;
-import static java.util.Collections.emptySet;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assume.assumeTrue;
-
-/**
- * This test verifies that SonarPython 1.13 gets excluded due to being below minimum supported version.
- */
-public class ConnectedModeExcludeByVersionTest extends AbstractConnectedTest {
```

Comment:
This IT was removed in the process, is it normal ?

Reply:
Yes, I think this test is not really interesting as an IT. We already have unit tests for that anyway. It is very costly to start a SQ server just for that old case IMO. My goal (see next PR) is to share instances of SQ as much as possible, and this might require dropping a few tests.